src/screens/nano/NanoContractDetail.js

```diff
@@ -144,6 +144,35 @@ function NanoContractDetail(props) {
     });
   }
 
+  const renderBalances = () => {
+    return Object.entries(ncState.balances).map(([token, data]) => {
+      return (
+        <tr key={token}>
+          <td>{data.value}</td>
+          <td>{token === hathorLib.constants.NATIVE_TOKEN_UID ? token : <Link to={`/token_detail/${token}`}>{token}</Link>}</td>
+        </tr>
+      );
+    });
+  }
+
+  const renderNCBalances = () => {
+    return (
+      <div className="table-responsive">
+        <table className="table table-striped table-bordered" id="attributes-table">
+          <thead>
+            <tr>
+              <th className="d-lg-table-cell">Amount</th>
+              <th className="d-lg-table-cell">Token</th>
```

Comment:
suggection(if-minor): Maybe we should invert these.

| Token | Amount |
| --- | --- | 
| 00 | 0.50 | 
| 00112233 | 3.00 |



| Amount | Token |
| --- | --- | 
| 0.50 | 00 | 
| 3.00 | 00112233 | 

The "token first" table seems more intuitive for the reader in my opinion.
What do you think?

Reply:
The first implementation had the token first but with the size of the Token UID I preferred having the amount first.

I will invert here again and validate my thoughts

Reply:
Done https://github.com/HathorNetwork/hathor-explorer/pull/287/commits/ca4491818941f89c536a414c372ebc777eea98a5

This commit also handles the decimal places to have the pretty value